network-api/networkapi/wagtailpages/views.py

```diff
@@ -17,7 +19,7 @@ def custom404_view(request, exception):
     belongs to which site, as "a site" is not tied to "a django
     app" in the wagtail way of things.
     """
-    if request.site.hostname == 'www.mozillafestival.org':
+    if Site.find_for_request(request).hostname == 'www.mozillafestival.org':
```

Comment:
as per Wagtail 2.9, see https://docs.wagtail.io/en/stable/releases/2.9.html#sitemiddleware-and-request-site-deprecated